rules/aep0004/resource_type_name.go

```diff
@@ -32,17 +32,17 @@ var resourceTypeName = &lint.MessageRule{
 	LintMessage: func(m *desc.MessageDescriptor) []lint.Problem {
 		resource := utils.GetResource(m)
 		_, typeName, ok := utils.SplitResourceTypeName(resource.GetType())
-		upperTypeName := utils.ToUpperCamelCase(typeName)
+		kebob_case := utils.ToKebobCase(typeName)
```

Comment:
```suggestion
		kebabCase := utils.ToKebobCase(typeName)
```
go variable naming is camelCase